tool/gravity/cli/register.go

```diff
@@ -207,7 +207,48 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.RestoreCmd.Timeout = g.RestoreCmd.Flag("timeout", fmt.Sprintf("Maximum time a restore job is active. Defaults to the value from the manifest or %v if unspecified", defaults.HookJobDeadline)).Duration()
 
 	// operations on gravity applications
-	g.AppCmd.CmdClause = g.Command("app", "operations on gravity applications")
+	g.AppCmd.CmdClause = g.Command("app", "Operations with application images and releases.")
+
+	// helm-specific flags
+	g.AppInstallCmd.CmdClause = g.AppCmd.Command("install", "This command installs an application from the specified application image.")
+	g.AppInstallCmd.Image = g.AppInstallCmd.Arg("image", "Specifies application image to install. Can be an image tarball, an unpacked image tarball, or an image name in the form of <name>:<version>.").Required().String()
+	g.AppInstallCmd.Name = g.AppInstallCmd.Flag("name", "Release name. If not specified, will be auto-generated.").String()
+	g.AppInstallCmd.Namespace = g.AppInstallCmd.Flag("namespace", "Namespace to install release into.").Default(defaults.Namespace).String()
+	g.AppInstallCmd.Set = g.AppInstallCmd.Flag("set", "Set values on the command line. Can specify multiple or comma-separated: key1=val1,key2=val2.").Strings()
+	g.AppInstallCmd.Values = g.AppInstallCmd.Flag("values", "Set values from the provided YAML file.").Strings()
+	g.AppInstallCmd.Registry = g.AppInstallCmd.Flag("registry", "Address of Docker registry to push application images to.").String()
+	g.AppInstallCmd.RegistryCA = g.AppInstallCmd.Flag("registry-ca", "Docker registry CA certificate path.").String()
+	g.AppInstallCmd.RegistryCert = g.AppInstallCmd.Flag("registry-cert", "Docker registry client certificate path.").String()
+	g.AppInstallCmd.RegistryKey = g.AppInstallCmd.Flag("registry-key", "Docker registry client private key path.").String()
+
+	g.AppListCmd.CmdClause = g.AppCmd.Command("ls", "This command shows all application releases.")
```

Comment:
nit: I think command description should not start with `This command` but simply state the purpose:
```go
"Show all application releases"
```